frontend/package.json

```diff
@@ -26,91 +26,93 @@
     "npm": ">=6.13.4"
   },
   "dependencies": {
-    "@github/markdown-toolbar-element": "^1.3.2",
+    "@github/markdown-toolbar-element": "^1.4.0",
     "@github/text-expander-element": "^2.2.0",
-    "@ungap/custom-elements": "^0.1.10",
+    "@ungap/custom-elements": "^0.1.15",
     "bem-react-helper": "^1.2.2",
     "classnames": "^2.2.6",
-    "core-js": "^3.8.2",
+    "core-js": "^3.9.0",
     "intersection-observer": "^0.12.0",
-    "lodash-es": "^4.17.20",
+    "lodash-es": "^4.17.21",
     "node-emoji": "^1.10.0",
-    "preact": "^10.5.9",
-    "react-intl": "^5.10.12",
+    "preact": "^10.5.12",
+    "react-intl": "^5.12.5",
     "react-redux": "^7.2.2",
     "redux": "^4.0.5",
     "redux-thunk": "^2.3.0"
   },
   "devDependencies": {
-    "@babel/preset-env": "^7.12.11",
-    "@babel/preset-react": "^7.12.10",
-    "@formatjs/cli": "^3.0.1",
+    "@babel/preset-env": "^7.12.17",
```

Comment:
Let's bump dependencies in a separate PR? It doesn't seem related to auth changes.

Reply:
Sure, I've already made a separate brand. PR will be in a min.